### ISSUE TITLE
HTTP 500 error ever since disabling stache watcher

### ISSUE THREAD

**Issue by DanjBethel** [statamic/cms#4873]
### Bug description

I added the STATAMIC_STACHE_WATER variable to my .env file and set it to false, tried going to my Collections and got a 500 error and this in the console "crbug/1173575, non-JS module files deprecated.". Tried adding it back but nothing fixes it. Weirdly enough, occasionally if I resave my .env file and refresh I will get the initial page load, but navigating away breaks it again.

### How to reproduce

I disabled the stache watcher.

### Logs

```shell
crbug/1173575, non-JS module files deprecated. in browser console.
```

### Versions

Statamic 3.2.25 Pro
Laravel 8.75.0
PHP 8.1.0
No addons installed

### Installation

Existing Laravel app

### Additional details

Peak starter-kit.

**Comment by DanjBethel** [comment]
I am also using Laravel Sail. Seeing a stacktrace error now:

Class "Illuminate\Log\LogManager" not found in /var/www/html/vendor/laravel/framework/src/Illuminate/Log/LogServiceProvider.php:17

**Comment by duncanmcclean** [comment]
Looks like this was a bug inside Laravel which has now been fixed (laravel/framework#40035). 

If you update to the latest version of Laravel, this should be resolved.
